### PR TITLE
apply gamemodes to a specific person

### PR DESCRIPTION
**Pull request by DeternoMC**
Please note I'm not a professional coder, however if you see a problem in my code, please tell me.

^ on that note, don't just merge make sure you read it. :)

### CODE REVIEW

**Comment by DeternoMC** [comment]
There's an error, I'm aware of one but not sure of the others.

File: src/main/java/net/frostedop/frostedopmod/commands/C_creative.java [hunk 1]
```diff
@@ -14,16 +14,28 @@
 public class C_creative extends FCommand {
 
     public C_creative() {
-        super("creative", "/creative", "", Arrays.asList("gmc"));
+        super("creative", "/creative <player>", Arrays.asList("gmc"));
```
Comment (Focusvity) [inline src/main/java/net/frostedop/frostedopmod/commands/C_creative.java]: Missing double `"` after usage.

File: src/main/java/net/frostedop/frostedopmod/commands/C_creative.java [hunk 2]
```diff
@@ -14,16 +14,28 @@
 public class C_creative extends FCommand {
 
     public C_creative() {
-        super("creative", "/creative", "", Arrays.asList("gmc"));
+        super("creative", "/creative <player>", Arrays.asList("gmc"));
     }
 
     @Override
-    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+    public boolean onCommand(CommandSender sender, Command cmd, Player playersender, String label, String[] args) {
```
Comment (Focusvity) [inline src/main/java/net/frostedop/frostedopmod/commands/C_creative.java]: This method `Player playersender` does not exist. Removal is required

File: src/main/java/net/frostedop/frostedopmod/commands/C_spectator.java [hunk 3]
```diff
@@ -12,15 +12,15 @@
  *
  * @author Savnith
  */
-// Make this command so admins can change OP's gamemode
+
 public class C_spectator extends FCommand {
 
     public C_spectator() {
-        super("spectator", "/spectator", "", Arrays.asList("spec"));
+        super("spectator", "/spectator <player>", Arrays.asList("spec"));
```
Comment (Focusvity) [inline src/main/java/net/frostedop/frostedopmod/commands/C_spectator.java]: Missing double `"` after usage.

File: src/main/java/net/frostedop/frostedopmod/commands/C_spectator.java [hunk 4]
```diff
@@ -12,15 +12,15 @@
...
 public class C_spectator extends FCommand {
 
     public C_spectator() {
-        super("spectator", "/spectator", "", Arrays.asList("spec"));
+        super("spectator", "/spectator <player>", Arrays.asList("spec"));
     }
 
     @Override
-    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+    public boolean onCommand(CommandSender sender, Command cmd, Player playerSender, String label, String[] args) {
```
Comment (Focusvity) [inline src/main/java/net/frostedop/frostedopmod/commands/C_spectator.java]: This method `Player playerSender` does not exist. Removal is required.

File: src/main/java/net/frostedop/frostedopmod/commands/C_survival.java [hunk 5]
```diff
@@ -14,16 +14,28 @@
 public class C_survival extends FCommand {
 
     public C_survival() {
-        super("survival", "/survival", "", Arrays.asList("gms"));
+        super("survival", "/survival <player>", Arrays.asList("gms"));
```
Comment (Focusvity) [inline src/main/java/net/frostedop/frostedopmod/commands/C_survival.java]: Missing double `"` after usage.

File: src/main/java/net/frostedop/frostedopmod/commands/C_survival.java [hunk 6]
```diff
@@ -14,16 +14,28 @@
 public class C_survival extends FCommand {
 
     public C_survival() {
-        super("survival", "/survival", "", Arrays.asList("gms"));
+        super("survival", "/survival <player>", Arrays.asList("gms"));
     }
 
     @Override
-    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
+    public boolean onCommand(CommandSender sender, Command cmd, Player playersender, String label, String[] args) {
```
Comment (Focusvity) [inline src/main/java/net/frostedop/frostedopmod/commands/C_survival.java]: This method `Player playerSender` does not exist. Removal is required.

**Comment by DeternoMC** [comment]
Not 100% sure if it's going to work, tell me if I need to add something!

**Comment by DeternoMC** [comment]
don't really think it will work .-. atleast i tried :P

**Comment by Focusvity** [comment]
I’ll patch it once I get home.

**Comment by DeternoMC** [comment]
Okay, shall I close it for now then?

**Comment by savnithyt** [comment]
Project uploaded, thank you for your PR's.

**Pull request closed without merging**